content/pt-br/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -0,0 +1,71 @@
+---
+reviewers:
+- 
+title: Política de Tráfego Interno do Serviço
```

Comment:
```suggestion
title: Política de Tráfego Interno do Service
```
?

Throughout this page (and its English original), Service is usually referring to `kind: Service` rather than to the general IT concept “service”.

Reply:
Hi @sftim! I understand your point but I think that if we do not translate Service the context will be strange.
IMO I'd like to keep the translation.
@edsoncelio Can you share you opinion here?
Tks

Reply:
Hi @sftim @devlware, i agree with your points, what you think if we use: `Política de Tráfego Interno do Recurso Service` or something like that? my point is to be more specific when we are talking about kubernetes resources names that can be translated in portuguese (and sometimes has another meaning).